addons/sale_management/tests/test_sale_order.py

```diff
@@ -53,6 +53,16 @@ def setUpClass(cls):
             ],
         })
 
+        cls.sale_order_option = cls.env['sale.order.option'].create({
+            'name': "Option",
+            'product_id': cls.optional_product.id,
+            'price_unit': 100,
+        })
+        cls.sale_order_with_option = cls.env['sale.order'].create({
+            'partner_id': cls.partner.id,
+            'sale_order_option_ids': cls.sale_order_option,
+        })
```

Comment:
If you add data only for a specific test, add it in your test, not in the class setup ;)

Reply:
yep, thank you :)